api/validators.py

```diff
@@ -105,10 +103,10 @@ def g(payload, method):
             try:
                 _validate_json(payload, _schema, resolver_input)
             except jsonschema.ValidationError as e:
-                handler.abort(400, str(e))
+                raise webapp2.exc.HTTPBadRequest(str(e))
```

Comment:
I'd prefer a little more encapsulation. Your code shouldn't expose its internals, such as its use of `jsonschema`. You should rather define a `ValidationException` so that I can then say `except validators.ValidationException`.